src/bindings/pygaia/classification/classificationtaskmanager.py

```diff
@@ -167,7 +167,9 @@ def mergeDataSet(self):
         log.info('Merging original base dataset...')
 
         from gaia2.fusion import mergeAll
-        mergeAll(self.conf['filelist'], self.conf['datasetFilename'], 5000, transfoFile = None)
+        # exclude tags fields as they may not be consistent among tracks
+        # and they don't contain relevant information for classification
+        mergeAll(self.conf['filelist'], self.conf['datasetFilename'], 5000, transfoFile = None, exclude = '.metadata.tags*')
```

Comment:
I am just thinking this hardcoded exclude value should be actually read from the classification project file.